### DISCUSSION TITLE
How to correctly set the resource path in template function's logo parameter?

### DISCUSSION THREAD

**Discussion by psgg1981** [hyperdiv/hyperdiv#62] [Help] [answered]
Hi,

I've reused the app template demo showcased in `hyperdiv docs`, however, when setting up my own project directories the app logo doesn't render using `template = hd.template(logo="./assets/hd-logo-white.svg", title="My App")`

```
.
├── start.py
└── myapp
    ├── assets
    │   └── hd-logo-white.svg
    ├── login.py
    ├── app.py
    └── users_db.py
```

...anything else missing?

Thanks!

**Comment by mariusnita** [comment]
Hi, that folder structure looks ok to me. I think the `.` in `"./assets/hd-logo-white.svg"` is likely problematic.

Try without the dot:

```py
hd.template(logo="/assets/hd-logo-white.svg", title="My App")
```

**Reply by psgg1981** [reply]
Thanks for the quick reply Marius, but it doesn't seem to work:

![image](https://github.com/hyperdiv/hyperdiv/assets/885067/111fb511-0b29-441b-932b-947650a84aec)

I tried other approaches as well e.g. `"./myapp/assets/hd-logo-white.svg"` but it didn't work either. The latter making sense after checking the output of `sys.modules['__main__']` since it returns `/home/.../projects/hyperdiv-test/start.py`

**Comment by mariusnita** [comment, marked as answer]
Ooh oops sorry. I think the problem is likely that the assets directory should be at the same level as `start.py`.

```
.
├── assets
│   └── hd-logo-white.svg
├── start.py
└── myapp
    ├── login.py
    ├── app.py
    └── users_db.py
```
